src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
+    }
+
+    /**
+     * @return string
+     */
+    public function sku() : string
+    {
+        return $this->sku;
+    }
+
+    /**
+     * @return int
+     */
+    public function quantity() : int
+    {
+        return $this->quantity;
+    }
+
+    /**
+     * @return string
+     */
+    public function cartId() : string
```

Comment:
1. noun - message "give me value". verb - message "do something" 
2. It is commonly user convention _introduced by doctrine and other tools_
3. I would avoid to use tools that force on me naming convention without any reason.